trackintel/preprocessing/positionfixes.py

```diff
@@ -214,6 +218,10 @@ def generate_triplegs(pfs_input, stps_input, method="between_staypoints", gap_th
     # copy the original pfs for adding 'staypoint_id' column
     pfs = pfs_input.copy()
 
+    # if the positionfixes already have a column "tripleg_id", we drop it
+    if "tripleg_id" in pfs:
```

Comment:
The original bug probably would not occur in this function: We wrote `pfs["tripleg_id"] = pd.NA` in Line [264](https://github.com/mie-lab/trackintel/blob/ddd0a9df13d5e4e0ffe962ccf261655e1c814aa4/trackintel/preprocessing/positionfixes.py#L264).

But it won't hurt to add this either :)

Reply:
noted :) I have retained this